charts/redpanda/templates/statefulset.yaml

```diff
@@ -140,13 +141,13 @@ spec:
                 rpk --config "$CONFIG" redpanda config set redpanda.rack "${RACK}"
                 {{- end }}
               {{- end }}
+          securityContext: {{ include "container-security-context" . | nindent 12 }}
```

Comment:
NIT:
Weird that we have almost in all places `nindent 10` for container security content. Should we remove one level of indentation in this file for all containers?

Reply:
This is the container securityContext. The `nindent 10` ones are Pod securityContext.

Reply:
I find it just as weird that go doesn't indent lists but just about every yaml linter expects it. I think indented lists are more readable.

Since this list is already indented, let's decide this outside this PR.